onnxruntime/test/providers/cpu/math/element_wise_ops_test.cc

```diff
@@ -3941,6 +3942,30 @@ TEST(MathOpTest, ErfMoreData) {
   test.Run();
 }
 
+TEST(MathOpTest, ErfCheckMultiThreadDataChunking) {
+  OpTester test("Erf", 9);
+  static constexpr int64_t size = 100000;
+  std::vector<float> inputs(size);
+  std::vector<float> outputs(size);
+
+  float low = -5.0f, high = 5.0f;
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_real_distribution<float> dist(low, high);
```

Comment:
consider using the `RandomValueGenerator` class. it shows the random seed on failure so failures are easier to reproduce.

https://github.com/microsoft/onnxruntime/blob/af4bf436d70f62afe486bb5b9d9cc7c8f3b5b958/onnxruntime/test/common/random_generator.h#L43

also consider using a fixed random seed to avoid flakiness in CI builds.